eng/tools/azure-sdk-tools/ci_tools/parsing/parse_functions.py

```diff
@@ -693,7 +705,7 @@ def get_version_py(setup_path: str) -> Optional[str]:
 
     # Find path to _version.py recursively
     for root, dirs, files in os.walk(file_path):
-        dirs[:] = [d for d in dirs if d not in EXCLUDE and not d.endswith(".egg-info")]
+        dirs[:] = [d for d in dirs if d not in EXCLUDE and not d.endswith(".egg-info") and not d.startswith(".")]
```

Comment:
so we dont see .venv_pylint got it

Reply:
Yeah. We will revisit if we ever have an official module start with `.` I guess? Haha